src/status_im/commands/handlers/jail.cljs

```diff
@@ -23,12 +23,11 @@
         (dispatch [:set-chat-ui-props {:validation-messages markup}]))
 
       result
-      (let [command' (assoc command :handler-data returned)
-            params'  (assoc params :command command')] 
-        (dispatch [:chat-send-message/send-command chat-id params']))
+      (let [command' (assoc command :handler-data returned)]
+        (dispatch [:chat-send-message/send-command (assoc params :command command')]))
```

Comment:
`let` can be removed here and `assoc-in` could be used instead:
`(assoc-in params [:command :handler-data] returned)`

Reply:
:+1: